documentation/lifecycle.md

```diff
@@ -40,8 +40,8 @@ const loader = container.resolve(LoaderService);
 loader
     .bootstrap(ClientModule)
     .afterComplete(() => {
-        // This method is called, if the lifecycle finished
-        // You can send event to server to inform the player is ready
+        // This method is called, if the lifecycle is finished
+        // You can send a event to server to inform the player is ready
```

Comment:
`a` to `an`

Reply:
@AlteredCarbonate first line, **your** must be **our**. it is our need :D